media/css/firefox/features/quantum/common.scss

```diff
@@ -0,0 +1,140 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../pebbles/includes/lib';
+@import '../../../hubs/mixins';
+@import '../../../hubs/sections';
+@import '../../../hubs/masthead';
+@import '../../../hubs/header';
+@import '../../../hubs/sub-nav';
+@import '../../../quantum/common';
+@import '../../../quantum/buttons';
```

Comment:
Funny story - I was originally told the button on `/new` would be a one-off special case, but it's already sneaking in to other places unsuprisingly. I don't think it's going to be the new default button style though (at least the one on `/firefox/` is more akin to the regular buttons and suits a white background). I don't suggest we do anything about this now, but it's something I think we might need to try and sort out with our designers :/

Reply:
😒